File: pass-gen.py
# Imports the string and random modules
import string, random
# Prints the title
print("== PASS-GEN ==")
# Asks the user for a desired password length and then converting it to an integer
length = int(input("[+] Desired password length: "))
# Initial values for the variables
index = 0
final_pass = ""
# This is a loop that will continue until the index is greater than the length
while index <= length:
    final_pass = final_pass + list(string.printable)[random.randint(0, 99)]
    index += 1
# Replaces the spaces in the final password with nothing, and then prints the final password
print("\nPassword:\n"+final_pass.replace(" ",""))

File: pass-gen.cpp
#include <iostream>
#include <string>
int main() {
  std::cout << "== PASS-GEN ==\n";
  std::cout << "[+] Desired password length: ";
  int password_length;
  std::cin >> password_length;
  std::string final_pass;
  char chars[] = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789!/*-+.&'(-_)=^$ù*:;,}]@^`|[{#";
  for (int index = 0; index <= password_length; index++) {
    final_pass = final_pass + chars[rand() % 91];
  }
  std::cout << final_pass;
  return 0;
}